depends/patches/capnp/abi_placement_new.patch

```diff
@@ -0,0 +1,68 @@
+commit 3007ac1e4a4bf80420471be45e25b3694025becd
```

Comment:
In commit "depends: patch around PlacementNew issue in capnp" (fe283860b10709ef8e7a69269c70708cb85f47f3)

This patch seems right but I can't figure out how it was generated. Commit `3007ac1e4a4bf80420471be45e25b3694025becd` does not seem to be a commit in the capnproto repository. The upstream patch is https://github.com/capnproto/capnproto/commit/74560f26f75dda4257dce541ca362a1e763b2971, which edits `c++/src/kj/common.h` and is applied with `-p2` while the patch here edits `src/kj/common.h` and is applied with `-p1`.

This all seems ok but it might be nice to use the upstream patch (can be generated with `git format-patch -n1 74560f26f75dda4257dce541ca362a1e763b2971`) to make the origin of the change clearer.

Reply:
Thanks, I've adjusted this to use `-p2` (should have just done this in the first place rather than adjusting the paths).